src/components/Dropdown/Dropdown.stories.tsx

```diff
@@ -9,11 +9,23 @@ const meta = {
   title: 'Components/Dropdown',
   component: Dropdown,
   parameters: {
-    layout: 'centered'
+    docs: {
+      story: {
+        height: '20rem'
+      }
+    }
```

Comment:
do we want to move this into a separate PR along with other components for ADS-730?